Portfolio setup : Header and Headline Section

NoelLincoln opened the pull request:
Make page responsive for devices within the range of 375px and 768px
Use percentages in width of elements in divs instead of pixels 
Set dimensions for logo

NoelLincoln committed:
Build header section for mobile:
    add a container div and set display property to flex
    add a div for logo and mobile menu inside the container
    design the logo and menu section using figma
    add menu icon svg and logo svg files to assets directory

NoelLincoln committed:
Fix on font error:
    correct misplaced double quote that brought up an error

NoelLincoln committed:
Fix:
    Add alt tag that was missing in the logo

NoelLincoln committed:
Add headline section :
    Add a container div for the heading tag and introduction text
    Add new logo and styling for mobile menu navigation
    Add media query for mobile devices and desktops
    Rearrange items in flex container and remove unnecessary divs

NoelLincoln committed:
Fix social media icons and headline background
    use individual vectors for each icon and add styling
    reduce logo size
    set background in headline section to cover and no-repeat

NoelLincoln committed:
Fix social media icons and headline background
    use individual vectors for each icon and add styling
    reduce logo size
    set background in headline section to cover and no-repeat

NoelLincoln committed:
Add text to headline section header

NoelLincoln committed:
FIx on responsiveness:
    Change px values to percentages to allow for responsiveness
    Use media query to add breakpoints for mobile and desktop screens
    Place nav inside header tag
    use logos width and height to remove overflow

NoelLincoln committed:
Fix linter errors on css file

NoelLincoln committed:
remove inline css on logo image

wayungi commented on index.html:
```diff
@@ -5,11 +5,61 @@
+        <nav class="navbar">
+          <!-- <div class="toolbar"> -->
+          <div class="logo">
+            <a href="">
```
- [ ] Please do not leave the `href=" "` blank. You can link it to the homepage. This is for 3 main reasons
  -  Convention
  - Semantics &
  - User experience. You can read more about it [here](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page)

wayungi replied:
- [OPTIONAL]  I think you should not include your logo in the nav tag. The `nav` tag should include the site's navigation menu. Both the logo & the nav however should be included in the `header` tag. 😄

wayungi commented on index.html:
```diff
@@ -5,11 +5,61 @@
-      <a href=""><logo>Noel Bryant</logo></a>
-    </header>
+    <div class="headline-container">
+      <header>
```
- [ ] The project requirement states that you should align both sections with Flexbox. The sections referred to here are the toolbar (header) and headline. Please double-check the[ project requirements.](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md)
![inst](https://github.com/NoelLincoln/mportfolio/assets/99656687/ee12dea5-8210-46d5-9350-a3ce700d2356)

wayungi commented on style.css:
```diff
@@ -0,0 +1,229 @@
+    flex-shrink: 0;
+    flex-direction: column;
+    align-items: flex-start;
+    margin-top: 20px;
+    padding: 114px 24px;
```
- [ ] The space between the header and headline is too large. Kindly reduce it to match the Figma

**Figma**
![scrn2](https://github.com/NoelLincoln/mportfolio/assets/99656687/6686a28f-eab5-4439-8b98-b70e2dacb146)

**Your design**
![scrn](https://github.com/NoelLincoln/mportfolio/assets/99656687/d2464125-0270-4575-ac39-071668523891)
.

wayungi commented on index.html:
```diff
@@ -5,11 +5,61 @@
+          <p>Let's connect</p>
+        </div>
+        <ul class="social-media-icons">
+          <li class="twitter">
+            <a href="https://twitter.com/Noel_Lincoln">
+              <img src="assets/images/social-media-icons/twitter.svg"
+                /></a>
+          </li>
+          <li class="linkedin">
+            <a href="https://www.linkedin.com/in/noel-bryant/">
+              <img src="assets/images/social-media-icons/linkedin.svg"
+                /></a>
+          </li>
+
+          <li class="github">
+            <img src="assets/images/social-media-icons/github.svg" />
+          </li>
+        </ul>
```
- [ ] You only have 3 social icons. The Figma has 5. Please add the missing 2 to match the Figma design. 👍🏾

wayungi commented on index.html:
```diff
@@ -5,11 +5,61 @@
+          me.
+        </p>
+        <div class="social-media">
+          <p>Let's connect</p>
```
- [ ] This is a link. Please use the anchor tag. This is because all links in HTML are anchor tags and apart from being semantically correct, the `a` tag will let you connect to other sections of the website, using href. 😺

wayungi reviewed (changes requested):
Hi @NoelLincoln , 

![image](https://user-images.githubusercontent.com/99656687/234198503-c41d0396-d5b0-4466-8ed1-959de33796fa.png)


**Congratulations on the good job done so far!** 🤝 

There are some issues that you still need to work on to go to the next project but you are almost there! 😄 

## Highlights ⚡ ⚡ 
- [ ] All linter checks are passing ✔️ 
- [ ]  Correct GitHub flow ✔️ 
- [ ] Built 2 sections of the portfolio ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

NoelLincoln committed:
Fix: Align elements in Header and Headline section
    Place navbar and logo separately under the header and align using flex
    Change position of header from relative to absolute
    Make all links point to a location or reseource
    Add missing social media icons

NoelLincoln committed:
Fix linter errors: remove 0 values

NoelLincoln commented:
Hello @wayungi ,

I have made all the required changes as advised.

LionRouge1 reviewed (approved):
Hi @NoelLincoln, 

# STATUS: APPROVED! 🥇 💯 
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

![done](https://media.giphy.com/media/a5llhdWKMOIfneMCpB/giphy.gif)

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
